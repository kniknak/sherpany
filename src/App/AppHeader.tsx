import React, { memo } from "react";
import { Link } from "react-router-dom";
import AppBar from "@material-ui/core/AppBar";
import Toolbar from "@material-ui/core/Toolbar";
import Button from "@material-ui/core/Button";

export const AppHeader: React.FC = memo(() => {
    return (
        <AppBar position="static">
            <Toolbar>
                <Button
                    color="inherit"
                    component={Link}
                    to="/"
                >
                    Home
                </Button>
                <Button
                    color="inherit"
                    component={Link}
                    to="/settings"
                >
                    Settings
                </Button>
            </Toolbar>
        </AppBar>
    );
});
