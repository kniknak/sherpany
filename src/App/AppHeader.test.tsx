import React from "react";
import { shallow } from "enzyme";
import { AppHeader } from "./AppHeader";

describe("AppHeader", () => {
    it("renders", () => {
        const subject = shallow(<AppHeader/>);

        expect(subject).toMatchSnapshot();
    });
});
