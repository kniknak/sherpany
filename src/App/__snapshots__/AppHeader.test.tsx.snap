// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppHeader renders 1`] = `
<WithStyles(ForwardRef(AppBar))
  position="static"
>
  <WithStyles(ForwardRef(Toolbar))>
    <WithStyles(ForwardRef(Button))
      color="inherit"
      component={
        Object {
          "$$typeof": Symbol(react.forward_ref),
          "displayName": "Link",
          "propTypes": Object {
            "innerRef": [Function],
            "onClick": [Function],
            "replace": [Function],
            "target": [Function],
            "to": [Function],
          },
          "render": [Function],
        }
      }
      to="/"
    >
      Home
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      color="inherit"
      component={
        Object {
          "$$typeof": Symbol(react.forward_ref),
          "displayName": "Link",
          "propTypes": Object {
            "innerRef": [Function],
            "onClick": [Function],
            "replace": [Function],
            "target": [Function],
            "to": [Function],
          },
          "render": [Function],
        }
      }
      to="/settings"
    >
      Settings
    </WithStyles(ForwardRef(Button))>
  </WithStyles(ForwardRef(Toolbar))>
</WithStyles(ForwardRef(AppBar))>
`;
